src/OceanSeaIceModels/InterfaceComputations/atmosphere_ocean_fluxes.jl

```diff
@@ -36,7 +36,8 @@ function compute_atmosphere_ocean_fluxes!(coupled_model)
     interface_properties = coupled_model.interfaces.atmosphere_ocean_interface.properties
     ocean_properties = coupled_model.interfaces.ocean_properties
     atmosphere_properties = (thermodynamics_parameters = thermodynamics_parameters(atmosphere),
-                             surface_layer_height = surface_layer_height(atmosphere))
+                             surface_layer_height = surface_layer_height(atmosphere),
+                             gravitational_acceleration = coupled_model.interfaces.gravitational_acceleration)
```

Comment:
we could consider `interface.properties`. Or maybe we should sketch out a larger vision for this.

Reply:
we could start with `interface.properties`.

Reply:
or `OceanSeaIceModel.properties` perhaps, which maybe makes more logical sense (even though the properties are _only_ needed for coupling)

Reply:
Actually `OceanSeaIceModel.interfaces` is mostly an organization principle, since we don't have the concept of a model that doesn't have interfaces